builtin/settingtypes.txt

```diff
@@ -204,6 +207,10 @@ keymap_rangeselect (Range select key) key KEY_KEY_R
 #    See http://irrlicht.sourceforge.net/docu/namespaceirr.html#a54da2a0e231901735e3da1b0edf72eb3
 keymap_freemove (Fly key) key KEY_KEY_K
 
+#    Key for toggling pitch fly mode.
+#    See http://irrlicht.sourceforge.net/docu/namespaceirr.html#a54da2a0e231901735e3da1b0edf72eb3
+keymap_freemove (Pitch fly key) key KEY_KEY_L
```

Comment:
`keymap_pitchfly` :)

Reply:
of course :)

Reply:
Would you please be so nice and update the README file as well?